test/fixtures/api_actions.yml

```diff
@@ -100,4 +100,12 @@ create_api_action_plugin_subdomain_events:
   api_namespace: plugin_subdomain_events
   http_method: post
 
+create_custom_api_action_one:
+  type: CreateApiAction
+  action_type: custom_action
+  include_api_resource_data: false
+  payload_mapping:
+  method_definition: ""
```

Comment:
@alis-khadka you should include a basic example of something here, like writing to the logs (API action response logs) or something like that